pysqldb3/geopackage.py

```diff
@@ -75,8 +83,32 @@ def write_gpkg(self, tbl_name = None, print_cmd=False):
         if not self.path:
             self.path = file_loc('folder')
 
-        if not self.table:
-            self.table = tbl_name
+        if not self.gpkg_tbl:
+            self.gpkg_tbl = self.table
+
+        # overwrite vs update vs an issue has arisen
+        if overwrite:
```

Comment:
maybe check if it exists 1st then if not you can skip the rest and it will simplify the other conditions

Reply:
If it exists, I don't want to automatically rewrite the file in case someone didn't realize they already had a geopackage with that name and had meant to rename their new file as something else.

Reply:
Ah sorry, my previous comment is confusing.  `overwrite` refers to the table, but the command has to be very specific based on what you want to do, which is why the `if` statement has 3 options. Following from my previous comment, I didn't want to accidentally overwrite a table in an existing package if someone meant to rename it as something else. But maybe you don't think that is a common-enough issue to merit that functionality?

In your example where a gpkg exists but a certain table doesn't, you can't write the same command as if the geopackage doesn't exist. you have to explicitly include the term "-update" in order to append the table to the geopackage. Otherwise, it will overwrite your entire geopackage and include only the 1 table (which I could see as a huge headache if you spent time compiling/curating tables for the gpkg). The `if` statement currently checks for all those scenarios.

Reply:
oh ok i get it i do think true false is more intuitive